### PR TITLE
Squash commits on setup

### PR DESCRIPTION
**Pull request by joshuaogle**

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
 # Set up NPM dependencies
-echo -e "Installing Dependencies..."
+echo "Installing Dependencies..."
 npm install
 
 # Remove Git remote if it's still the proteus-middleman repo
 if [ `git config --get remote.origin.url` = "git@github.com:thoughtbot/proteus-gulp.git" ]; then
-  echo -e "What is your repo url? Enter URL or leave blank"
+  echo "What is your repo url? Enter URL or leave blank"
   read url
```
Comment (gabebw) [inline bin/setup]: quote `"$url"` on the lines below this one

File: bin/setup [hunk 2]
```diff
@@ -2,23 +2,23 @@
...
 # Run this script immediately after cloning the codebase.
 
-#!/usr/bin/env sh
-
-# Run this script immediately after cloning the codebase.
+# Squash all commits to get a clean slate
+git update-ref -d refs/heads/master
+git commit -am "New Proteus-Jekyll project"
 
 # Make sure Bundler is installed
```
Comment (gabebw) [inline bin/setup]: Should this comment say NPM, not bundler?

**Comment by joshuaogle** [comment]
Thanks @gabebw. Fixed the quotes and Bundler -> npm

File: bin/setup [hunk 3]
```diff
@@ -2,26 +2,26 @@
...
 npm install
 
 # Remove Git remote if it's still the proteus-middleman repo
-if [ `git config --get remote.origin.url` = "git@github.com:thoughtbot/proteus-gulp.git" ]; then
-  echo -e "What is your repo url? Enter URL or leave blank"
+if [ "$(git config --get remote.origin.url)" = "git@github.com:thoughtbot/proteus-gulp.git" ]; then
+  echo "What is your repo url? Enter URL or leave blank"
   read url
-  if [ ! $url = '' ]; then
+  if [ ! "$url" = "" ]; then
```
Comment (gabebw) [inline bin/setup]: There is actually a built-in way to check for empty strings!

```bash
if [ -z "$url" ]; then
```

More here: http://tldp.org/LDP/abs/html/comparison-ops.html

**Pull request merged**